src/components/CalculatorKeys.js

```diff
@@ -1,30 +1,45 @@
+import PropTypes from 'prop-types';
 import CalculatorDisplay from './CalculatorDisplay';
 
-function CalculatorKeys() {
+function CalculatorKeys({
+  btnClick, total, operation, next,
+}) {
   return (
     <div className="wrapper">
-      <CalculatorDisplay />
-      <div className="btn light-gray">AC</div>
-      <div className="btn light-gray">+/-</div>
-      <div className="btn light-gray">%</div>
-      <div className="btn orange">÷</div>
-      <div className="btn light-gray">7</div>
-      <div className="btn light-gray">8</div>
-      <div className="btn light-gray">9</div>
-      <div className="btn orange">X</div>
-      <div className="btn light-gray">4</div>
-      <div className="btn light-gray">5</div>
-      <div className="btn light-gray">6</div>
-      <div className="btn orange">+</div>
-      <div className="btn light-gray">1</div>
-      <div className="btn light-gray">2</div>
-      <div className="btn light-gray">3</div>
-      <div className="btn orange">-</div>
-      <div className="btn btn-0 light-gray">0</div>
-      <div className="btn light-gray">.</div>
-      <div className="btn orange">=</div>
+      <CalculatorDisplay total={Number(total)} operation={operation} next={next} />
+      <button type="button" className="btn light-gray" onClick={btnClick}>AC</button>
+      <button type="button" className="btn light-gray" onClick={btnClick}>+/-</button>
+      <button type="button" className="btn light-gray" onClick={btnClick}>%</button>
```

Comment:
- [ ] Your modulus works great except when the number should yield 0, in that case, it doesn't display anything, kindly update this 
https://www.awesomescreenshot.com/video/17939255?key=f308c86da819dd2bea750f6f7e93c0f5